### DISCUSSION TITLE
troubles with configuring the app

### DISCUSSION THREAD

**Discussion by Jooosl** [lollipopkit/flutter_server_box#567] [Q&A] [answered]
the documentation is a bit lacking, so i don't quite understand how everything works, i hope this is the right place to ask.
i installed the docker image and tunneled it through nginx. i want to use the widget on my phone and also a web access on my pc.
- what features do i get exactly?
- how do i configure it right? (what do i put in YOUR_TOKEN in the config file?) (all i get is the message "not found")

thank you for your help!

**Comment by lollipopkit** [comment, marked as answer]
> what features do i get exactly?

- msg push (server cpu usage...)
- home widget status

> how do i configure it right?

for the config file: https://github.com/lollipopkit/server_box_monitor/wiki#fill-server-config
